Game.h

```diff
@@ -98,9 +98,18 @@ class Game
     void prepareScene(RGB={255/3,255/3,255/3});
     void presentScene();
     // control logic (game.cpp)
+    constexpr static uint32_t DELAY = 16;
     void initSystems();
     void processInput();
     void gameLoop();
+    // timer logic
+    int level = 1;
+    int timeLeft = 0;
+    bool fastFall = false;
+    // dropDelay is in terms of DELAY increments.
+    int dropDelay() const {
```

Comment:
`dropDelay` is actually incorrect, I believe. There should be some conversion from 'G' (which is what the `dropDelay` formula uses) to ms, frames, or seconds, which would allow for a more faithful simulation.